app/frontend/src/cljs/teet/contract/contract_responsibilities_view.cljs

```diff
@@ -2,63 +2,105 @@
+                        (conj
+                          acc
+                          [[[url/Link (:navigation-info item)
+                             (tr-enum (:task/type item))]]
+                           [(:owner item)]
+                           [(if (nil? (:task/assignee item))
+                              ""
+                              (user-model/user-name (:task/assignee item)))]
+                           [(tr-enum (:task/status item))]]))
+                      []
+                      tasks)]]
+     [:tbody
+      [:tr
+       [:th {:colSpan 4
+             :class (<class contract-style/responsibilities-table-header-style)}
+        (tr-enum title)]]
+      (ui-util/mapc
+        (fn [row]
+          [:tr {:class (<class contract-style/responsibilities-table-row-style)}
+           (ui-util/mapc
+             (fn [[column {:keys [style class] :as opts}]]
+               [:td (merge
+                      {:style (merge {} style)
+                       :class (herb/join
+                                (<class contract-style/responsibilities-table-cell-style)
+                                (when class
+                                  class))}
+                      (dissoc opts :style :class))
+                column])
+             row)])
+        rows)])])
 
-(defn targets-responsibilities-table
+(defn targets-responsibilities
   [targets]
   [:div
-   [table/simple-table
-    [[]]
-    (for [target targets
-          :let [task? (some? (get-in target [:target :task/type]))
-                groups (group-by :task/group (get-in target [:target :activity/tasks]))]]
-      [[(get-in target [:project :thk.project/name])]
-       [(tr [:enum (get-in target [:target :activity/name])])]
-       [[url/Link
-         (merge (:target-navigation-info target)
-                {:component-opts {:data-cy "target-responsibility-activity-link"}})
-         (tr [:link :target-view-activity])]
-        ]])]])
+   (for [target targets
+         :let [task? (some? (get-in target [:target :task/type]))
+               tasks (get-in target [:activity :activity/tasks])
+               groups (group-by :task/group tasks)]]
+     [:div {:style {:padding "2rem 0 2rem 0"}}
+      [:div {:class (herb/join (<class common-styles/flex-row-w100-space-between-center)
+                               (<class common-styles/margin-bottom 2))}
+       [typography/Heading1 (get-in target [:project :thk.project/name])]
+       [typography/Heading4 (if (get-in target [:target :activity/name])
+                              (tr [:enum (get-in target [:target :activity/name])])
+                              "")]
+       [url/Link
+        (merge (:target-navigation-info target)
+               {:component-opts {:data-cy "target-responsibility-activity-link"}})
+        (tr [:link :target-view-activity])]]
+      (when (not-empty groups)
+        (simple-table-with-many-bodies [[(tr [:contract :responsible :header :task])]
```

Comment:
Is there a reason this is a function call (`(simple-table-with-many-bodies ...`), not a component (`[simple-table-with-many-bodies ...`)? Not that it matters AFAIK since the data is static for the lifetime of the parent component.